app/controllers/users_controller.rb

```diff
@@ -9,24 +9,22 @@ def index
 
   # GET /users/1
   # GET /users/1.json
-  def show
-  end
+  def show; end
 
   # GET /users/new
   def new
     @user = User.new
   end
 
   # GET /users/1/edit
-  def edit
-  end
+  def edit; end
 
   # POST /users
   # POST /users.json
   def create
     @user = User.new(user_params)
 
-    respond_to do |format|
+    respond_to do |_format|
```

Comment:
Rubocop told you to put an underscore before this variable because you're not using it. But, if you're not using this variable, maybe just remove that block.